assets/src/BlockEditorValidation.js

```diff
@@ -22,7 +23,9 @@ export const blockEditorValidation = () => {
     }
 
     const hasImageInContent = /<img.+wp-image-(\d+).*>/i.test(postContent);
-    if (!featuredImage && !hasImageInContent) {
+    // If postType === 'wp_block' it's a pattern creation, and these don't have featured images.
+    const needsFeaturedImage = postType !== 'wp_block' && !featuredImage && !hasImageInContent;
```

Comment:
What if an NRO website creates a new post type without featured images?
Maybe, instead of checking in this condition that the is different than `wp_block` we could check for an array of post types that require a featured image.
What do you think?

Reply:
That's a good point! I'll change the implementation 👍